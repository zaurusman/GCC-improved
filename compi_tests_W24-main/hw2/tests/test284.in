return			

;	
  	
 
  	 
	return;  		
	
   	 	 	 	
continue;
break;
continue;
booly =not2b		
 
+ 	
 		
	
	(z)	

 		 ;
byte
	  a= true
		
	



		
			;if("a wizard"
		  			
	
	
	 	 
				  
	
	 	

	
  	


   
 >"you're")if	(
	 "harry")
z=false;//Yoursexlifeshouldgobacktofirstgrade.  
		
 	
 
		^^
i=  	

 	  



 		j *

 false and falseor true		 

			  *not




 

	 	
	 	"you're"	
	 ;//Yourfood issmallerthen requiredto


passthese

  tests.mother fucker...j("harry")	
	 		

 	
	;//Yourcode		 isunreadable.^^inti

;//Yourgithub lookslikeapiazza box.	<3 	 
  
 
 				

		 	

 

 

	   
	 	
 	continue;	 

 
  
 	
	if  
	 (j(z 
 	or 
 
 
	7
 	

	

  	 	 			 
				
and	false / 3b*false/s4ElKk*false-	i	/	true+z
* 1	+ 3  	 	 	
 b+ BPmFyEb+	"\n"/j 
or	  "a wizard"+5 b+8 and true) )z 
=i; elsex	   	 
	
(j);x(0 b+		

		
 
	
	
 	 
 	  		
"a wizard"*a or( 				 
2
 
  	 
	

	or		
	
		    
	
  	 
	
		  
 

 
 		
   
    
 	1

	

b) 	



 
 

	 
 	
	 	-

"harry")	 		

  	
	;//Your partner
		is unreadable.:)	 	  		 		

 	 	


	 
  	 
 		     boola;
 	 
 	

 	  	 	 	
	  	
	 		
	
 

		
 	//Your  
  				



	  


  mama should		 

 
		
 


	 


			
	 
 	   
 	   
  
go
backtofirst grade. uwu
z(a( 

  (int) falseandy or not j (true)	/5bor7-7- y87

	 			 	


  	

 (z)
or Skmoh5ortrue+
 	  	 
	  
(	
   

 
8 b	 	 		 
   	 

	

		
	

		

  			

		
 
	 

 	* (	 

	
not 0b)	 
		 


  	    		  		)>= 	



 ( 	
	  
 
	


	 	
 

	 			
  
	 
	 
	 
		 		 	 byte )"you're"*2 *
 
	 
		

true and"a  	

wizard" * (x) 
	

 -8b -false	   <=0b) + true 
 			 	
 
 
	/ not(
j) );		 			 	
		
	 

break ;	 
 continue;	 				

if(7		  	 )byteRrp= 	   	 
  	 
 
   						 (byte	
		  
)2b;
elsebool y;bool  		y ="e";break		;while(		


	 
	 	 		  a)	
{a =a(GYx("harry")	 	);
bytex;
return;
	

		    


 		
	 return;
	
	 		 	 		 
		break;	
}if ( x){i	(z
(false))

	;
	 		 	} 	 elsereturn;//Your 

laptop	

 isun	readable.mother
fucker...j= 	( i
		  	 
	


    )	 ;		 
	

 	
		 



	
	//Yourppissmaller then  required to passthesetests. ^^
y=
z;byte
 
 
 		
   	

 
	

y=not
( bool)false
;//Yourcode makes recruters startsmoking :)
 			
 if


( 
 81  	


	

					  
)
j((
		 byte )5	
  	 b
	
 	
 )		 		; 
	else
inqtz;

			     return;int l3iDEMby	

 =z  ;//Your mama is noneexistent...{	
return; 
 
{while

    
	
	
	 	

  
 	 ( 8
  
  



  


	
	
 	
	)	bool  	
 	    
	 i= 3; 		i(true)  

 

; }				

 	 	


//Your code
  makesrecrutersstartsmoking:	)y(31#381059609
   b)
	

 			 	 	 


 	  

	 
  ;i="you're" 
	 	  * 7	   		and		 	
	


 
 

		 	  
	
  			    	 
			
  	 

  		true
	
   
  ;//Your sex
lifeso 5FAT wh9enOshe jumps intothe

ocean<theoceanjumpsout.				  	  mot(herfucker... 	
 	 	

  	
}
if(		 
  6b )	 if  (
  
 
	z)	

 x("you're" 


 	 	/i)	
			   	   
	
 
   	
	 	 
 		
	
 ;//Yourpart&ner isun
readable. uwureturn;//Yourfoodis 
none	existent.uwu
uc2I9zvOhx

	 
 	=9 
;return 

 	;	
		y ("e" 	
);"

 