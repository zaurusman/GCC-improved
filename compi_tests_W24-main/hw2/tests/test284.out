9: Statement -> RETURN SC
2: Statements -> Statement
9: Statement -> RETURN SC
3: Statements -> Statements Statement
14: Statement -> CONTINUE SC
3: Statements -> Statements Statement
13: Statement -> BREAK SC
3: Statements -> Statements Statement
14: Statement -> CONTINUE SC
3: Statements -> Statements Statement
21: Exp -> ID
21: Exp -> ID
19: Exp -> LPAREN Exp RPAREN
20: Exp -> Exp BINOP Exp
7: Statement -> ID ASSIGN Exp SC
3: Statements -> Statements Statement
17: Type -> BYTE
26: Exp -> TRUE
6: Statement -> Type ID ASSIGN Exp SC
3: Statements -> Statements Statement
25: Exp -> STRING
25: Exp -> STRING
31: Exp -> Exp RELOP Exp
25: Exp -> STRING
27: Exp -> FALSE
7: Statement -> ID ASSIGN Exp SC
line 49: lexical error
